src/docs/devices/ESP32andmore/ECB

```diff
@@ -0,0 +1,43 @@
+---
+title: ESP32andmore ESP32 Control Board
+type: sensor
+standard: global
+board: esp32
+project-url: https://github.com/fpovoski/ESP32-Temperature-Monitoring-PWM-Control-Board
+difficulty: 1
+made-for-esphome: true
```

Comment:
```suggestion
made-for-esphome: false
```

As stated here https://github.com/esphome/esphome-devices/pull/791#discussion_r1697796206 this device is not a part of this program and should be false.